packages/google_sign_in/google_sign_in_ios/ios/Classes/FLTGoogleSignInPlugin.h

```diff
@@ -6,5 +6,13 @@
 
 #import "messages.g.h"
 
+#import <GoogleSignIn/GoogleSignIn.h>
+
 @interface FLTGoogleSignInPlugin : NSObject <FlutterPlugin, FSIGoogleSignInApi>
+
+// Configuration wrapping Google Cloud Console, Google Apps, OpenID,
+// and other initialization metadata.
+// @property(strong) GIDConfiguration *configuration;
+@property(strong, nonatomic) GIDConfiguration *configuration;
```

Comment:
I moved this here so that I could access it in tests to validate it was set correctly. Let me know if there's a better way.